index.html

```diff
@@ -3127,6 +3133,208 @@ <h5>JWT Decoding</h5>
+            </tr>
+            <tr>
+              <td>
+A way to associate datatypes with values such as dates and times.
+              </td>
+              <td class="missing">✖</td>
+              <td class="supported">✓</td>
+              <td class="supported">✓</td>
+            </tr>
+            <tr>
+              <td>
+A facility to express one or more directed graphs, such as a social network,
+in a single document.
+              </td>
+              <td class="missing">✖</td>
+              <td class="supported">✓</td>
+              <td class="supported">✓</td>
+            </tr>
+            <tr>
+              <td>
+Supports signature sets.
+              </td>
+              <td class="missing">✖</td>
+              <td class="supported">✓</td>
+              <td class="supported">✓</td>
+            </tr>
+            <tr>
+              <td>
+Embeddable in HTML such that search crawlers will index the machine-readable
+content.
+              </td>
+              <td class="missing">✖</td>
+              <td class="missing">✖</td>
+              <td class="supported">✓</td>
+            </tr>
+            <tr>
+              <td>
+Data on the wire is easy to debug.
+              </td>
+              <td class="missing">✖</td>
+              <td class="missing">✖</td>
+              <td class="supported">✓</td>
+            </tr>
+
+            <tr>
+              <td>
+Nesting signed data does not cause data size to double for every embedding.
+              </td>
+              <td class="missing">✖</td>
+              <td class="missing">✖</td>
+              <td class="supported">✓</td>
+            </tr>
+
+            <tr>
+              <td>
+Proof format supports Zero-Knowledge Proofs.
+              </td>
+              <td class="missing">✖</td>
+              <td class="missing">✖</td>
+              <td class="supported">✓</td>
+            </tr>
+
+            <tr>
+              <td>
+Proof format supports arbitrary proofs such as Proof of Work, Proof of
+Elapsed Time, and Proof of Stake.
+              </td>
+              <td class="missing">✖</td>
+              <td class="missing">✖</td>
+              <td class="supported">✓</td>
+            </tr>
+
+            <tr>
+              <td>
+Proofs can be expressed unmodified in other data syntaxes such as
+YAML, N-Quads, and CBOR.
+              </td>
+              <td class="missing">✖</td>
+              <td class="missing">✖</td>
+              <td class="supported">✓</td>
+            </tr>
+
+            <tr>
+              <td>
+Changing key-value ordering, or introducing whitespace does not
+invalidate signature.
+              </td>
+              <td class="missing">✖</td>
+              <td class="missing">✖</td>
+              <td class="supported">✓</td>
+            </tr>
+
+            <tr>
+              <td>
+Support experimental signature systems.
+              </td>
+              <td class="missing">✖</td>
```

Comment:
Of course this is possible in JWT as well

Reply:
Based on your input, the language has been changed to this:

> PF16: Designed to easily support experimental signature systems.

with the following explanation:

> The LD-Proof format is naturally extensible, not requiring the format to be
extended in a formal international standards working group in order to
prevent namespace collisions. The JWT format requires entries in a centralized
registry in order to avoid collisions and does not support experimentation as easily as the LD-Proof format does. LD-Proof format extension is done through the decentralized publication
of cryptographic suites that are guaranteed to not conflict with other LD-Proof
extensions. This approach enables developers to easily experiment with new
cryptographic signature mechanisms that support selective disclosure,
zero-knowledge proofs, and post-quantum algorithms.